src/senaite/core/skins/senaite_templates/senaite_widgets/recordswidget.pt

```diff
@@ -98,7 +100,7 @@
                                 id string:${fieldName}-${key}-${repeat/idx/index};
                                 value python:field.getSubfieldValue(values, idx, key);
                                 size python:field.getSubfieldSize(key);
-                                maxlength python:field.getSubfieldMaxlength(key);
+                                maxlength maxlength|Nothing;
```

Comment:
Not sure if this works with capital `N`. Please use `|nothing` with lower case `n`.

Reply:
It works with `Nothing` too. But yes, better to keep consistency